src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -504,6 +550,16 @@ public class GenerateProtoTask extends DefaultTask {
     }
   }
 
+  @Nested
+  protected Collection<PluginOptions> getBuiltinsInternal() {
+    return builtins
+  }
+
+  @Nested
+  protected Collection<PluginOptions> getPluginsInternal() {
```

Comment:
I suppose these two getters for exposing the two containers as properties so that they can be validated, since there public equivalents cannot be called after configuration is finished. A comment to explain that would be helpful.